temporalio/bridge/src/runtime.rs

```diff
@@ -100,6 +126,20 @@ impl Runtime {
     }
 }
 
+#[pymethods]
+impl RuntimeRef {
+    fn retrieve_buffered_metrics<'p>(&self, py: Python<'p>) -> Vec<BufferedMetricUpdate> {
+        convert_metric_events(
+            py,
+            self.runtime
+                .metrics_call_buffer
+                .as_ref()
+                .unwrap()
```

Comment:
There's enough unwrapping going on in this callstack it might make sense to just actually have this return a Result

Reply:
I considered that, but I decided to trust the caller. In theory the Python code will never call this unless it was setup with a buffer. So there isn't a user-facing failure here that should happen.

Reply:
Added `expect` to clarify the case, but I never expect this `unwrap` to fail